### PR TITLE
(#119) Wildcard imports removed

### PR DESCRIPTION
**Pull request by AsadNizami**
Fix for #193. I have tested all the examples. Sorry for working on the issues before it's assigned to me.

### CODE REVIEW

**Review by bhosale2 — APPROVED** [review]
LGTM.

**Comment by AsadNizami** [comment]
Sure, I'll get it done!

**Comment by AsadNizami** [comment]
Please review.

**Comment by bhosale2** [comment]
Hi @AsadNizami I had 2 comments:

1. Please check the formatting failing tests and fix them.
2. I saw that you got rid of the `__all__` keywords from submodules. However if you see the `__init__.py` in `elastica`, the wildcard imports now import everything from individual submodules, which is not required and is also a bit inefficient. Can you please remove those wildcards, and add specifically import the functionalities we need there from the submodules? If you have any questions about this let me know.

**Comment by AsadNizami** [comment]
> 2\. specifically import the functionalities we need there from the submodules?

This includes all the classes mentioned in the `__init__` of all submodules and those of other files present in the same directory as `elastica/__init__.py`, right?

**Comment by bhosale2** [comment]
@AsadNizami  Yes, the idea is wherever possible wildcard imports should be replaced while making sure existing example scripts and paths don't break.

**Comment by AsadNizami** [comment]
Pls review!

File: elastica/wrappers.py [hunk 1]
```diff
@@ -1,19 +1,5 @@
...
-    "Forcing",
-    "CallBacks",
-    "Damping",
-]
-from elastica.modules.base_system import BaseSystemCollection
-from elastica.modules.connections import Connections
-from elastica.modules.constraints import Constraints
-from elastica.modules.forcing import Forcing
-from elastica.modules.callbacks import CallBacks
-from elastica.modules.damping import Damping
```
Comment (bhosale2) [inline elastica/wrappers.py:16]: we may need to keep this for backward compatibility. @AsadNizami what was the reasoning to remove this?

**Review by bhosale2 — CHANGES_REQUESTED** [review]

File: elastica/wrappers.py (hunk 1 above)
Comment (AsadNizami) [reply elastica/wrappers.py:16]: > Please check the formatting failing tests and fix them.

image


The unused import statements were causing the formatting test to fail.

Comment (AsadNizami) [reply elastica/wrappers.py:16]: We can mention the imports in the `__all__` keyword to fix the issue.

Comment (bhosale2) [reply elastica/wrappers.py:16]: I see thanks @AsadNizami for the clarification. Let's bring back the `__all__` then for backward compatibility. There may still be some users that refer to the `modules` as `wrappers`.

Comment (AsadNizami) [reply elastica/wrappers.py:16]: Sure thing!

**Comment by AsadNizami** [comment]
Pls review.

**Review by bhosale2 — APPROVED** [review]
LGTM. @armantekinalp this one is all yours.

**Review by armantekinalp — APPROVED** [review]
LGTM. Thanks

**Pull request merged**